### PR TITLE
Added base features

### PR DESCRIPTION
**Pull request by neharajesh**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { HomeScreen } from "./components/HomeScreen";
+import { LoginScreen } from "./components/Login/LoginScreen";
+import "./styles.css"
+import { Route, Routes } from "react-router";
+import { Page } from "./components/Page";
+import { VideoPlayer } from "./components/VideoPlayer";
+
```
Comment (DanishOnGit) [inline src/App.js:7]: u can group them in an index file ....
then will look like this `import {Page, VideoPLayer} from "./components"`

File: src/components/HomeScreen.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+export const HomeScreen = ({sidebar}) => {
+    return(<>
+    <div className="container-homescreen">
+        <CategoriesBar />
+        <div className="container-video flex flex-row-wrap">
+            {videoData.map((video) => (
+                <div key={video.id}
+                className="video-card card-w-25 h-auto mg-l-1 csr-point txt-s pd-b-05 mg-tb-05 flex-col flex-col-space-evenly">
+                    <div className="video-top mg-b-05 card-h-10" >
+                        <img className="w-100 h-auto" src={`https://img.youtube.com/vi/${video.id}/hqdefault.jpg`} alt="Thumbnail" />
```
Comment (DanishOnGit) [inline src/components/HomeScreen.js:15]: are u using some css library?

File: src/styles.css [hunk 3]
```diff
@@ -0,0 +1,284 @@
...
+    padding: 0 1.25rem;
+    color: var(--color-text);
+    background: transparent;
+    border: none; 
+}
+
+.header-avatar {
+    border-radius: 50%;
+    height: 40px;
+    width: 40px;
```
Comment (DanishOnGit) [inline src/styles.css]: we can use rem here too

File: src/styles.css [hunk 4]
```diff
@@ -0,0 +1,284 @@
...
+
+.sidebar li {
+    display: flex;
+    align-items: center;
+    padding: 0.5rem 1.5rem;
+    margin: 0.2rem 0;
+    cursor: pointer;
+}
+
+.sidebar li>span {
```
Comment (DanishOnGit) [inline src/styles.css:170]: this has high specificity. Use class on the `span` instead

File: src/styles.css [hunk 5]
```diff
@@ -0,0 +1,284 @@
...
+    .sidebar {
+        width: 90px;
+    }
+    .sidebar li {
+        justify-content: center;
+    }
+    .sidebar li>span {
+        display: none;
+    }    
+}
```
Comment (DanishOnGit) [inline src/styles.css:197]: same here

File: src/styles.css [hunk 6]
```diff
@@ -0,0 +1,284 @@
...
+
+@media screen and (max-width: 600px) {
+    .sidebar {
+        transform: translateX(-120%);
+        position: fixed;
+        z-index: 999;
```
Comment (DanishOnGit) [inline src/styles.css]: add this z-index too to your variables list

**Review by DanishOnGit — COMMENTED** [review]
Hi Neha,
 I have suggested minor suggestions as of now, mostly css

File: src/App.js (hunk 1 above)
Comment (neharajesh) [reply src/App.js:7]: I'll do this! thanks! :D

File: src/components/HomeScreen.js (hunk 2 above)
Comment (neharajesh) [reply src/components/HomeScreen.js:15]: it's my component library!

File: src/styles.css (hunk 3 above)
Comment (neharajesh) [reply src/styles.css]: will do!

File: src/styles.css (hunk 4 above)
Comment (neharajesh) [reply src/styles.css:170]: okay, will do that!

File: src/styles.css (hunk 6 above)
Comment (neharajesh) [reply src/styles.css]: okay!

File: src/components/HomeScreen.js (hunk 2 above)
Comment (DanishOnGit) [reply src/components/HomeScreen.js:15]: ohh alright !

File: src/components/Header.js [hunk 7]
```diff
@@ -0,0 +1,20 @@
+import { FaBars } from 'react-icons/fa';
+import { FiSearch } from "react-icons/fi";
+import { FcVideoProjector } from "react-icons/fc";
+import avatar from "../images/avatar.jpg"
+import "../styles.css"
+
+export const Header = ({handleToggleSidebar}) => {
+    return (<>
+        {console.log("from header")}
```
Comment (DarkShadow20) [inline src/components/Header.js]: You should remove console.log

File: src/styles.css [hunk 8]
```diff
@@ -0,0 +1,284 @@
...
+        display: none;
+    }
+}
+
+/* SIDEBAR */
+.sidebar {
+    background-color: var(--color-secondary-black);
+    display: flex;
+    flex-direction: column;
+    width: 200px;
```
Comment (DarkShadow20) [inline src/styles.css:139]: you should use consistent units for the css properties

**Review by DarkShadow20 — CHANGES_REQUESTED** [review]

File: src/components/Header.js (hunk 7 above)
Comment (neharajesh) [reply src/components/Header.js]: will do! thanks!

File: src/styles.css (hunk 8 above)
Comment (neharajesh) [reply src/styles.css:139]: will do thanks! :D

**Comment by neharajesh** [comment]
Implemented all changes

**Pull request merged**